src/libs/StepperMotor.h

```diff
@@ -69,6 +71,7 @@ class StepperMotor  : public Module {
         float steps_per_mm;
         float max_rate; // this is not really rate it is in mm/sec, misnamed used in Robot and Extruder
         float acceleration;
+        float pressure_advance;
```

Comment:
I would like to avoid adding this for every motor as it uses valuable memory for a feature rarely used. and will hurt 6 axis machines which would not even be using extruders.